app/services/database_service.py

```
@@ -423,126 +390,203 @@ def list_content_items(
+
+    def save_yt_comment(self, comment_data: dict) -> Optional[dict]:
+        """Insert a new YouTube comment record."""
+        if not self.is_available:
+            return None
+        try:
+            with get_session() as session:
+                obj = YouTubeComment(**comment_data)
+                session.add(obj)
+                session.flush()
+                return obj.to_dict()
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)